libs/dh/market-participant/feature-organization/src/lib/dh-market-participant-organization.component.html

```diff
@@ -0,0 +1,135 @@
+<!--
+@license
+Copyright 2020 Energinet DataHub A/S
+
+Licensed under the Apache License, Version 2.0 (the "License2");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<ng-container
+  *transloco="let transloco; read: 'marketParticipant.organization'"
+>
+  <table
+    mat-table
+    [dataSource]="this.dataSource"
+    class="mat-elevation-z8"
+    aria-label="organizations"
+  >
+    <!-- Organization Name -->
+    <ng-container matColumnDef="{{ this.columnIds[0] }}">
+      <th mat-header-cell *matHeaderCellDef>
+        {{ transloco('columns.organization') }}
+      </th>
+      <td mat-cell *matCellDef="let element">
+        {{ element.organization.name }}
+      </td>
+    </ng-container>
+
+    <!-- Actor GLN -->
+    <ng-container matColumnDef="{{ this.columnIds[1] }}">
+      <th mat-header-cell *matHeaderCellDef>
+        {{ transloco('columns.gln') }}
+      </th>
+      <td mat-cell *matCellDef="let element">
+        <div *ngIf="element.actor !== undefined">
+          {{ element.actor.gln.value }}
```

Comment:
Consider: Is `element.actor.gln` always set? Otherwise be careful about deeply nested objects. You can use the safe navigation operator to be avoid issues like this `element?.actor?.gln?.value`. Also if some of the values can be null or undefined consider if fallback values should be shown. (this affects all the values in the template)

Reply:
GLN is always set, while actor can be missing. I've added fallbacks in places.